modules/map-canvas.js

```diff
@@ -67,7 +406,8 @@ class MapDialog extends FormApplication {
             scaleControl: true,
             disableDefaultUI: false,
             streetViewControl: false, // TODO: Figure out how to make Street View capture properly.
-            mapTypeId: google.maps.MapTypeId[game.settings.get("map-canvas", "DEFAULT_MAP_MODE")],
+            //mapTypeId: google.maps.MapTypeId[game.settings.get("map-canvas", "DEFAULT_MAP_MODE")],
+            mapTypeId: google.maps.MapTypeId.SATELLITE,
```

Comment:
Is there a specific reason the default mode needs hard coded to SATELLITE?

Reply:
You are right -it doesnt need to be; I was creating a lot of maps and it was driving me crazy - maybe it could be a setting.

Let me know if you want me to revert this line ir if you want to do it.

Reply:
Thanks ! I made the changes. Love the module; Perfect  for superhero RPGs